utils.py

```diff
@@ -232,7 +233,11 @@ def createVar():
       if [ -f sql/mysqld ]; then extra="$extra sql/mysqld"; fi
       if [ -f sql/mariadbd ]; then extra="$extra sql/mariadbd"; fi
     fi
-    tar zcf var.tar.gz mysql-test/var/*/log mysql-test/var/log ${extra}
+    tar zcf var.tar.gz mysql-test/var/*/log/*.err mysql-test/var/log
```

Comment:
err files are already in `<a href="https://ci.mariadb.org/%(prop:tarbuildnum)s/logs/%(prop:buildername)s/">mysqld* log dir</a>`.

Asside from that `*.err` is the wrong glob, and `mysql-test/var/log` is the singe mtr test run case.

This removes the datadir of the failed test case so inspection of that isn't possible.